command/agent/command.go

```diff
@@ -169,10 +156,36 @@ func (c *Command) readConfig() *Config {
 	f.StringVar(&retryIntervalWan, "retry-interval-wan", "",
 		"Time to wait between join -wan attempts.")
 
+	// deprecated flags
+	var atlasJoin bool
+	var atlasInfrastructure, atlasToken, atlasEndpoint string
+	f.StringVar(&atlasInfrastructure, "atlas", "",
+		"(deprecated) Sets the Atlas infrastructure name, enables SCADA.")
+	f.StringVar(&atlasToken, "atlas-token", "",
+		"(deprecated) Provides the Atlas API token.")
+	f.BoolVar(&atlasJoin, "atlas-join", false,
+		"(deprecated) Enables auto-joining the Atlas cluster.")
+	f.StringVar(&atlasEndpoint, "atlas-endpoint", "",
+		"(deprecated) The address of the endpoint for Atlas integration.")
+
 	if err := c.Command.Parse(c.args); err != nil {
 		return nil
 	}
 
+	// check deprecated flags
```

Comment:
no since the ones will be for the HTTP endpoint and the other ones for the command line. I'll add the `==> ` prefix though.